modules/custom/openy_search/openy_google_search/openy_google_search.module

```diff
@@ -82,13 +85,21 @@ function openy_google_search_help($route_name, RouteMatchInterface $route_match)
  */
 function openy_google_search_openy_search_theme_configuration_alter(&$overrides) {
   $search_config = \Drupal::configFactory()->get('openy_google_search.settings');
-  $search_page_id = $search_config->get('search_page_id');
-  $langcode = Drupal::languageManager()->getCurrentLanguage()->getId();
-  $alias = ltrim(Drupal::service('path.alias_manager')->getAliasByPath('/node/' . $search_page_id, $langcode), '/');
+  $alias = ltrim(_get_alias(), '/');
```

Comment:
Do we really need `ltrim()` here? Can `_get_alias()` return an alias with space at the beginning of a string?

Reply:
Fixed